packages/commonjs/test/types.ts

```diff
@@ -14,7 +14,8 @@ const config: import("rollup").RollupOptions = {
 			ignoreGlobal: false,
 			sourceMap: false,
 			namedExports: { './module.js': ['foo', 'bar' ] },
-			ignore: [ 'conditional-runtime-dependency' ]
+			ignore: [ 'conditional-runtime-dependency' ],
+      dynamicRequireTargets: [ 'node_modules/logform/*.js' ]
```

Comment:
spacing.

Reply:
It's weird some code in the project has tabs, some spaces. Here it's tabs.